weather_mv/loader_pipeline/ee.py

```diff
@@ -788,6 +801,11 @@ def start_ingestion(self, asset_data: AssetData) -> t.Optional[str]:
                 logger.info(f"Failed to create asset '{asset_name}' in earth engine: {e}. Moving on...")
                 return ""
 
+            error_text = repr(e).lower()
+            if "property" in error_text or "type does not match" in error_text:
```

Comment:
nit: 
if "type does not match the type of the same property of existing assets in the same collection" in repr(e):